backup/Parser1.java

```diff
@@ -0,0 +1,29 @@
+public class Parser {
+
+    public Command handleInput(String input) {
+        String[] data = input.split(" ");
+        switch data[0] {
+            case "list":
```

Comment:
I love the usage of switch cases. Prevents repeated if else statements, or what we call the arrowhead. Take note of switch case indentations.